project/deps.sc

```diff
@@ -7,9 +7,10 @@ object Scala {
   def scala213        = "2.13.14"
   def runnerScala3    = "3.0.2" // the newest version that is compatible with all Scala 3.x versions
   def scala3LtsPrefix = "3.3"   // used for the LTS version tags
-  def scala3Lts    = s"$scala3LtsPrefix.3" // the LTS version currently used in the build
-  def scala3Next   = "3.4.2"               // the newest/next version of Scala
-  def scala3NextRc = "3.5.0-RC7"           // the latest RC version of Scala Next
+  def scala3Lts  = s"$scala3LtsPrefix.3" // the LTS version currently used in the build
+  def scala3Next = "3.5.0"               // the newest/next version of Scala
+  def scala3NextAnnounced = "3.4.2"     // the newest/next version of Scala that's been announced
```

Comment:
As some tests need a version available for installation on `coursier`, we should bump this separately after 3.5.0 gets announced.